### PR TITLE
Add ParagraphPreprocess event

### PR DESCRIPTION
**Pull request by guiajlopes**

### CODE REVIEW

File: src/Event/Preprocess/Variables/ParagraphEventVariables.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ */
+class ParagraphEventVariables extends AbstractEventVariables {
+
+  /**
+   * @return \Drupal\paragraphs\Entity\Paragraph
+   */
+  public function getParagraph() {
+    return $this->variables['paragraph'];
+  }
+}
```
Comment (pdenooijer) [inline src/Event/Preprocess/Variables/ParagraphEventVariables.php]: Please add a line-break.

**Comment by pdenooijer** [comment]
Thank you for your contribution!
I'll try to merge it this weekend 🙂.

File: src/Event/Preprocess/Variables/ParagraphEventVariables.php (hunk 1 above)
Comment (guiajlopes) [reply src/Event/Preprocess/Variables/ParagraphEventVariables.php]: updated

**Review by pdenooijer — CHANGES_REQUESTED** [review]
Please extend Preproces unit tests with your new class 🙂

**Comment by pdenooijer** [comment]
Please rebase on the 8.x-1.x branch 🙂

**Review by pdenooijer — APPROVED** [review]

**Pull request merged**